tests/tests.js

```diff
@@ -13,11 +12,307 @@ const UNICODE_PATTERN = UNICODE_SET.toString();
 
 const IS_NODE_6 = process.version.startsWith('v6.');
 
+const FLAGS_WITH_UNICODE = 'u ui ug um uy uig uim uigm uigmy'.split(' ');
+const FLAGS_WITH_UNICODE_WITH_I = 'ui uig uim uigm uigmy'.split(' ');
+const FLAGS_WITH_UNICODE_WITHOUT_I = 'u ug um uy ugm ugmy'.split(' ');
+// Note: the leading space is important.
+const FLAGS_WITHOUT_UNICODE = ' i g m y ig im igm igmy'.split(' ');
+const FLAGS = FLAGS_WITH_UNICODE.concat(FLAGS_WITHOUT_UNICODE);
+
+const unicodeFixtures = [
```

Comment:
Can we still pull it into a separate files, that users can easily replace `regexpu-fixtures` with `regexpu-core/tests/regexpu-fixtures.js`?